NYPLCardCreator/Flow/AddressViewController.swift

```diff
@@ -361,19 +344,13 @@ final class AddressViewController: FormTableViewController {
             alternativeAddressesAndCardTypes: addressesAndCardTypes)
           self.navigationController?.pushViewController(viewController, animated: true)
         case .unrecognizedAddress:
-          let alertController = UIAlertController(
-            title: NSLocalizedString(
-              "Unrecognized Address",
-              comment: "An alert title telling the user their address was not recognized by the server"),
-            message: NSLocalizedString(
-              "Your address could not be verified. Please try another address.",
-              comment: "An alert message telling the user their address was not recognized by the server"),
-            preferredStyle: .alert)
-          alertController.addAction(UIAlertAction(
-            title: NSLocalizedString("OK", comment: ""),
-            style: .default,
-            handler: nil))
-          self.present(alertController, animated: true, completion: nil)
+          let title = NSLocalizedString(
+            "Unrecognized Address",
+            comment: "An alert title telling the user their address was not recognized by the server")
+          let message = NSLocalizedString(
+            "Your address could not be verified. Please try another address.",
```

Comment:
maybe this is for @killerpuppytails but this message seems a little jarring to me. Meaning, I entered my home address and perhaps I made a formatting mistake, but I may definitely not have "another address".

Reply:
I believe @killerpuppytails is away until next week, I will merge this PR for now and leave this until she is back.